### DISCUSSION TITLE
Has anyone successfully deploy to Caprover using GitHub Actions?

### DISCUSSION THREAD

**Discussion by dankore** [caprover/caprover#1013] [General]
I need help.

If anyone has deployed to Caprover from GitHub Actions and has a sample repo on GitHub please share it with me.

The less complicated configuration the better.

I need a sample repo or tutorial that shows how to set up `build and deploy ` jobs from a GitHub repo to Caprover.

What I know
- How to set up GitHub secrets
- How to set up a private registry on my Caprover

Any help or pointers would be greatly appreciated!

**Comment by githubsaturn** [comment]
Have you tried asking these folks?
https://github.com/marketplace/actions/caprover-deploy
https://github.com/marketplace/actions/caprover-app-deploy

**Reply by dankore** [reply]
I did look at those repos however I did not contact them.

I was able to get a repo that implemented a GitHub Action that deploys to Caprover.

Here's my version of it for anyone interested. 
https://github.com/dankore/deploy-to-caprover-using-github-actions

**Reply by maietta** [reply]
> I did look at those repos however I did not contact them.
> 
> I was able to get a repo that implemented a GitHub Action that deploys to Caprover.
> 
> Here's my version of it for anyone interested. 
> https://github.com/dankore/docker

Link doesn't work for me.

**Reply by dankore** [reply]
You can view it now!

I had it on Private visibility. Changed it to Public!

**Reply by dankore** [reply]
Updated repo URL to https://github.com/dankore/deploy-to-caprover-using-github-actions

**Comment by maietta** [comment]
Thanks!

- Nicholas
________________________________

**Comment by epam-vkerkhoff** [comment]
I find the way via a captain-definition file in a repo and a webhook much more convenient.

**Comment by lexoyo** [comment]
This is how I deploy [v3.silex.me](https://v3.silex.me) with github actions: https://github.com/silexlabs/editor.silex.me/blob/main/.github/workflows/prod.yml
